### ISSUE TITLE
🧪 Verify Admin side pages for any visual discrepancies

### ISSUE THREAD

**Issue by brindasasi** [GCTC-NTGC/gc-digital-talent#10798]
## 🧪 Tests

Describe the additions/changes you would like to see to the tests.

## 🦋 Expected Behaviour

Visual elements should look correct with the alignment , header, spacing etc., 

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] Go through each and every admin page and check on the visual elements including all the dialogs 
- [ ] Log the issues that you find

Out of scope 
- Fixing the issues is not part of this ticket. Just focus on testing. 

## 🛑 Blockers

Issues which must be completed before this one.

```[tasklist]
### Blocked By
- [ ] ticket number
```
